packages/blade/src/hooks/useDidUpdate.ts

```diff
@@ -0,0 +1,31 @@
+import type { DependencyList, EffectCallback } from 'react';
+import { useEffect, useRef } from 'react';
+
+// TODO: add tests
+
+/**
+ * Hook to run an effect on dependencies change after the component has *mounted*.
+ * Core difference from `useEffect` is this will trigger effects after the very first render.
```

Comment:
isn't it then same as running `useEffect` with `[]` as dependency?

Reply:
Quite the opposite, for this case I want to skip the effect for first time but run it for subsequent renders